app/shared/src/commonMain/kotlin/data/models/subject/SubjectProgressInfo.kt

```diff
@@ -58,12 +58,17 @@ data class SubjectProgressInfo(
             val lastWatchedEpIndex = episodes.indexOfLast {
                 it.type == UnifiedCollectionType.DONE || it.type == UnifiedCollectionType.DROPPED
             }
-
             val continueWatchingStatus = kotlin.run {
                 val latestEp = kotlin.run {
                     episodes.lastOrNull { it.isKnownCompleted }
                 }
 
+                var actualSubjectStarted = subjectStarted
+                // 有剧集 isKnownCompleted == true 时就认为已开播
+                if (latestEp != null) {
+                    actualSubjectStarted = true
+                }
```

Comment:
```suggestion
                // 有剧集 isKnownCompleted == true 时就认为已开播
                val actualSubjectStarted = latestEp != null || subjectStarted
```
一般尽可能避免 var, 会让逻辑更清晰可预测